e2e-tests/portal/cypress/integration/journeys/login/reset-password.spec.js

```diff
@@ -31,13 +31,6 @@ context('Reset password', () => {
 
       cy.url().should('eq', relative('/login?passwordreset=1'));
     });
-
-    it('should be case insensitive when accepting email', () => {
```

Comment:
We need this test here because the /reset-pssword route needs to be case insensitive

Reply:
Added.